src/shillelagh/backends/apsw/vt.py

```diff
@@ -191,6 +191,10 @@ def get_all_bounds(
         column_type = columns[column_name]
 
         # convert constraint to native Python type, then to DB specific type
+        if isinstance(constraint, set):
```

Comment:
might be to link to / paste the comment from here: https://rogerbinns.github.io/apsw/vtable.html#apsw.VTCursor.Filter

Reply:
done